comp2.html

```diff
@@ -0,0 +1,180 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
```

Comment:
It looks like you're mixing/matching indent depth here — should just be two spaces.